File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cpp
class Solution {
public:
    int longestConsecutive(std::vector<int> &nums)
{

    std::unordered_set<int> map(nums.begin(), nums.end());
    int maxLength = 0;

    for (const int &num : map)
    {

        if (map.find(num - 1) == map.end())
        {
            int current_length = 1;

            while (map.find(num + current_length) != map.end())
            {
                current_length++;
            }
            maxLength = std::max(maxLength, current_length);
        }
    }

    return maxLength;
}

};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
      
    if (nums == null || nums.length == 0)
      return 0;

    int slow = 0;
    int size = nums.length;
    for (int fast = 1; fast < size; fast++) {

      if (nums[slow] != nums[fast]) {
        slow++;
        nums[slow] = nums[fast];
        
      }

    }

    return slow + 1;

  
    }
}